### DISCUSSION TITLE
Pricing: Add Hobby Plan

### DISCUSSION THREAD

**Discussion by nahtnam** [supabase/supabase#10277] [Feature Requests]
It would be nice to have a "hobby" plan for something like $5-10 a month which has all the free plan limitations but allows you to create more apps. I find that a lot of my projects are small enough that I can't justify $25 a month minimum

**Comment by AWaselnuk** [comment]
I want to chime in and add I could _really_ use a plan for "DEV" instances (which are similar to hobby).

My setup is to have 1 DEV project and one PRODUCTION project in Supabase for every app. Now that I'm on to app number 2 with supabase I'm having some rough DX here. Whenever I switch which app I'm focused on I have to pause one and start up the other one.

If I could pay $5 a month to make the DEV projects never sleep I would absolutely do that. They could have limited usage much like a Hobby plan might have.
